nodejs/week1/week1homework/src/bakckend/index.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const app = express();
+const port=3000;
+
+const mealsRoute = require("./routes/meals")
+const aMealRoute = require("./routes/meal");
+const largeMealsRoute = require("./routes/large-meals");
+const cheapMealsRoute = require("./routes/cheap-meals");
+const reservationsRoute = require("./routes/reservations");
+const singleReservationRoute = require("./routes/reservation");
```

Comment:
By convention we put all the imports/requires at the top of a file, and the code below:

```
const express = require("express");
const mealsRoute = require("./routes/meals")
const aMealRoute = require("./routes/meal");
const largeMealsRoute = require("./routes/large-meals");
const cheapMealsRoute = require("./routes/cheap-meals");
const reservationsRoute = require("./routes/reservations");
const singleReservationRoute = require("./routes/reservation");

const app = express();
const port=3000;

// rest of the code
```

If you use a formatter this gets appropriately formatted for you with a good set of defaults. Companies will often have a formatter that is tied to their personal style guide.